cardano-testnet/src/Testnet/Property/Util.hs

```diff
@@ -76,3 +80,18 @@ runInBackground act = void . H.evalM $ allocate (H.async act) cleanUp
 decodeEraUTxO :: (IsShelleyBasedEra era, MonadTest m) => ShelleyBasedEra era -> Aeson.Value -> m (UTxO era)
 decodeEraUTxO _ = H.jsonErrorFail . Aeson.fromJSON
 
+isBootstrapPhase
```

Comment:
We can (and should) avoid this function totally by hardforking to PV10 instead. Adding a separate test that hardforks for PV9 would be more concise.

Reply:
Removed, made testnet start in PV10